tests/foreman/api/test_contentview.py

```diff
@@ -750,6 +751,105 @@ def test_composite_content_view_with_same_repos(self, module_org, target_sat):
         comp_content_view_info = comp_content_view.version[0].read()
         assert comp_content_view_info.package_count == 36
 
+    @pytest.mark.stream
+    @pytest.mark.tier2
+    def test_ccv_audit_scenarios(self, module_org, target_sat):
```

Comment:
Normally we name tests with `test_positive_...` or `test_negative_...`. Recently I've seen some stuff that doesn't do this convention. Are reviewers enforcing this naming standard? If so these can all be updated to `test_positive_...`.

Reply:
no, we no longer require test_positive. all tests should be assumed positive unless otherwise stated.

Reply:
Awesome!